src/Components/RestPassword/ResetInterface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Data{
+    email:string,
+    password:string,
+    cPassword?:string,
```

Comment:
You can consider change this name to a more meaningful name, as it is not clear what does the c in `cPassword` mean.